handlers/homu_status/__init__.py

```diff
@@ -2,6 +2,7 @@
 
 import json
 import re
+import HTMLParser
```

Comment:
This could just be `from HTMLParser import HTMLParser` and it could be moved to the previous group.